src/rule.js

```diff
@@ -10,9 +10,11 @@
         email: /^([\w-\.]+)@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.)|(([\w-]+\.)+))([a-zA-Z]{2,4}|[0-9]{1,3})(\]?)$/,
         number: /^\d+$/,
         minlength: function(value, scope, element, attrs, param) {
+          value += '';
```

Comment:
Ok, perfect. So Ill remove the string conversion, update the related tests and squash the commits to single one. Regarding the validation expression, your correct - it doesn't really matter now. 

The last thing: "can you remove those unwanted indent/format commit?" could you please specify 
more in detail which lines should be changed? (If not already discussed above) Thanks :)